### DISCUSSION TITLE
Adding a css just like getdoks.org home page

### DISCUSSION THREAD

**Discussion by raqueeb** [thuliteio/doks#443] [General]
You are an outstanding designer. I just wanted to understand where these files are located? I couldn't find it in https://github.com/h-enk/doks-gh-pages repository. Is https://getdoks.org/ hosted in GH pages?

1. 
![image](https://user-images.githubusercontent.com/32799073/132002647-d2d9a4a4-92ab-4285-9466-e1c296b8c30c.png)

2. 
![image](https://user-images.githubusercontent.com/32799073/132002749-509ca789-d63f-4c25-9585-49009ecc25f3.png)

Thank you in advance for the response.

**Comment by h-enk** [comment]
@raqueeb, the (indeed brilliant) design is by Tommy Vedvik, all the other work is by me.

These are sections of the getdoks.org homepage — see https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html:
1. [Newsletter section](https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html#L203-L211)
2. [Sponsors section](https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html#L134-L180)

The newsletter section calls a partial — https://github.com/h-enk/getdoks.org/blob/master/layouts/partials/sidebar/newsletter.html

A form submission is processed in https://github.com/h-enk/getdoks.org/blob/master/assets/js/app.js#L46-L59  — and uses a lambda function (https://github.com/h-enk/getdoks.org/blob/master/functions/submission-created.js) to submit to [Buttondown email](https://buttondown.email/)

**Reply by raqueeb** [reply]
Wow! That was quick! Now, it seems clear now. That's perfect!

Thank you once again.

> @raqueeb, the (indeed brilliant) design is by Tommy Vedvik, all the other work is by me.
> 
> These are sections of the getdoks.org homepage — see https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html:
> 
> 1. [Newsletter section](https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html#L203-L211)
> 2. [Sponsors section](https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html#L134-L180)
> 
> The newsletter section calls a partial — https://github.com/h-enk/getdoks.org/blob/master/layouts/partials/sidebar/newsletter.html
> 
> A form submission is processed in https://github.com/h-enk/getdoks.org/blob/master/assets/js/app.js#L46-L59 — and uses a lambda function (https://github.com/h-enk/getdoks.org/blob/master/functions/submission-created.js) to submit to [Buttondown email](https://buttondown.email/)
